packages/website/components/accountPlansModal/accountPlansModal.js

```diff
@@ -1,12 +1,60 @@
-import BillingPlanCards from '../account/billingPlanCards/billingPlanCards.js';
+import { useState } from 'react';
+import { Elements, ElementsConsumer } from '@stripe/react-stripe-js';
+
+import Loading from '../../components/loading/loading.js';
+import Button from '../../components/button/button.js';
+import CurrentBillingPlanCard from '../../components/account/currentBillingPlanCard/currentBillingPlanCard.js';
 import Modal from '../../modules/zero/components/modal/modal';
 import CloseIcon from '../../assets/icons/close';
-// import Button from '../button/button.js';
-import GradientBackground from '../gradientbackground/gradientbackground';
-import { plans } from '../contexts/plansContext';
+import AddPaymentMethodForm from '../../components/account/addPaymentMethodForm/addPaymentMethodForm.js';
+import { API, getToken } from '../../lib/api';
+
+export async function putUserPayment(pm_id, plan_id) {
+  const storage = plan_id ? { price: plan_id } : null;
+  const paymentSettings = {
+    paymentMethod: { id: pm_id },
+    subscription: { storage: storage },
+  };
+  const res = await fetch(API + '/user/payment', {
```

Comment:
This use of fetch seems to encode the same api request as the other PUT [over here](https://github.com/web3-storage/web3.storage/pull/1878/files#diff-7cc8b048a95005ae2552d91b2be79b342228050c6ef92a204229debb1039f951R12).

It seems like the pattern in this part of the codebase has been to define functions that request the http api in `/packages/website/lib/api.js` and then use them from places like this (and the above link).

@cmunns Will you update the files that trigger this PUT request to both invoke the same funtion in `api.js` that encapsulates how to do the request. That way the 'save payment settings' api request logic is only in once place (e.g. [changes like this i just did](https://github.com/web3-storage/web3.storage/pull/1878/commits/7849b53b7e81ad5621d6085911cc43fd3519046e))